nuxt.config.js

```diff
@@ -45,6 +46,13 @@ export default {
     // See https://github.com/nuxt-community/axios-module#options
   },
 
+  /*
+  ** ServerSide middleware
+  */
+  serverMiddleware: [
+    rateLimit({ windowMs: 3000, max: 3 })
```

Comment:
Please leave a comment here *why* that middleware is included

Reply:
Maybe a hint that ppl can/should remove that locally :thinking:

Reply:
Hmm... I don't know. What do you think of a `codesandbox` module inside repo?

Reply:
Nice idea! :+1: @pi0

Reply:
Done